ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource

```diff
@@ -566,11 +571,10 @@ Set Up Project
     ...                creates caikit runtime. This can be used as test setup
     [Arguments]    ${namespace}    ${single_prj}=${TRUE}    ${enable_metrics}=${FALSE}    ${dc_name}=kserve-connection
     IF    ${single_prj}
-        Clean All Models Of Current User
+        Run Keyword And Continue On Failure    Clean All Models Of Current User
     END
     Open Data Science Projects Home Page
-    Wait For RHODS Dashboard To Load    wait_for_cards=${FALSE}    expected_page=Data Science Projects
-    Create Data Science Project    title=${namespace}    description=kserve test project
+    Create Data Science Project    title=${namespace}    description=kserve test project    existing_project=${TRUE}
```

Comment:
This is the only line I'm not sure about, specifically `existing_project=${TRUE}`.
Could you test the change against `422__model_serving_caikit_py_client.robot`, `422__model_serving_llm_other_runtimes_UI.robot` and `422__model_serving_llm_UI.robot`?